drivers/input/input_ite_it8xxx2_kbd.c

```diff
@@ -16,38 +16,28 @@
 #include <zephyr/input/input.h>
 #include <zephyr/kernel.h>
 #include <zephyr/sys/atomic.h>
+#include "input_kbd_matrix.h"
```

Comment:
I think this header should be moved to include/, otherwise oot drivers won't be able to use this feature.

Reply:
Yes! Fixed :-) had that done already in https://github.com/zephyrproject-rtos/zephyr/pull/65117